### PR TITLE
add discord button and make buttongroup from styling components

### PR DESCRIPTION
**Pull request by Odyssey346**
Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

### CODE REVIEW

**Comment by Odyssey346** [comment]
so much bloat just to make it look good

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,37 @@
 import logo from './logo.svg';
 import './App.css';
+import styled from 'styled-components';
+
+const ButtonGroup = styled.div`
+  display: flex;
+`
```
Comment (Devnol) [inline src/App.js:7]: Unnecessary package, the same job can probably be accomplished with a div or some other css.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+        <ButtonGroup>
+          <a
+            className="Neutron-Tek"
+            href="https://github.com/neutrontechnologies"
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Check out our organization!"
+          >
+            <button>Organization page</button>
+          </a>
```
Comment (Devnol) [inline src/App.js:24]: <a> tag over the button just doesn't seem correct, also the way to assign classes in JSX is different.

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,37 @@
...
+        <p>Neutron Technologies</p>
+        <ButtonGroup>
+          <a
+            className="Neutron-Tek"
+            href="https://github.com/neutrontechnologies"
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Check out our organization!"
+          >
+            <button>Organization page</button>
```
Comment (Devnol) [inline src/App.js:23]: The React component for buttons is <Button>, this is the <button> html tag which doesn't have the react component features. (I know it was like that before, that's our issue).

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,37 @@
...
+          </a>
+          &nbsp;
+          <a
+            className="Neutron-Tek"
```
Comment (Devnol) [inline src/App.js:27]: That class name sounds stupid for what is a button and I have no idea why Midou added it in the first place.

**Review by Devnol — COMMENTED** [review]
Fix issues and I'll review it again.

File: src/App.js (hunk 2 above)
Comment (Odyssey346) [reply src/App.js:24]: Midou did this.

File: src/App.js (hunk 1 above)
Comment (Odyssey346) [reply src/App.js:7]: True.

File: src/App.js (hunk 4 above)
Comment (Odyssey346) [reply src/App.js:27]: yes.

**Review by Devnol — COMMENTED** [review]
I think it looks good, but I'll merge it once CI is ready. If possible also pull the changes to .gitignore from the dev branch

**Comment by Odyssey346** [comment]
merge now pls

**Comment by Odyssey346** [comment]
> I think it looks good, but I'll merge it once CI is ready. If possible also pull the changes to .gitignore from the dev branch

I've pulled to my dev-branch, however, to my patch-* branches I don't have to because I dispose of them once they get closed/merged.

**Comment by Devnol** [comment]
Okay lgtm imma merge now

**Comment by Odyssey346** [comment]
lgtm to me too

**Pull request merged**